src/interpreter/index.ts

```diff
@@ -1650,6 +1663,40 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
         return value;
     }
 
+    debugLocalVariables(): string {
```

Comment:
Could you add a jsdoc comment explaining the purpose of this function? It feels like an internal-to-brs function but I'm not sure what it's for just looking at it.

Reply:
I will add, this is a method to show the Variables on the REPL interface (TTY) but also will be used when I integrate the MicroDebugger